node/src/main/resources/reference.conf

```
@@ -1,6 +1,6 @@
 // a path to a working directory, where all keys, configurations and codes files will be stored
 // it's usually set by entrypoint.sh
-root-path = "/master"
+root-path = "target/.tendermint"
```

is it possible to delete this for fail fast on deploy mistakes?
if it needed for tests, add this into test config.